srcpkgs/deadbeef/template

```diff
@@ -16,19 +16,36 @@ depends="desktop-file-utils hicolor-icon-theme"
 short_desc="Ultimate Music Player for GNU/Linux"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="Zlib, GPL-2.0-or-later, LGPL-2.1-or-later"
+_apbuild_commit=c64f490e54447e3902fae54784d01ab59fa7dc95
+_mp4p_commit=f111d7aa10d4516f5e02465245e9202db83b7c10
 homepage="http://deadbeef.sourceforge.net"
 changelog="http://deadbeef.sourceforge.net/news0.html"
-distfiles="https://github.com/DeaDBeeF-Player/deadbeef/archive/${version}.tar.gz>${pkgname}-${version}.tar.gz"
-checksum=f239af734dea096083e9fb907ef262099a54c2c6c93683ed76b132d41f10ccd8
-build_options="gtk3"
-build_options_default="gtk3"
+distfiles="https://github.com/DeaDBeeF-Player/deadbeef/archive/${version}.tar.gz>${pkgname}-${version}.tar.gz
+ https://github.com/DeaDBeeF-Player/apbuild/archive/${_apbuild_commit}.tar.gz>apbuild-${_apbuild_commit}.tar.gz
+ https://github.com/DeaDBeeF-Player/mp4p/archive/${_mp4p_commit}.tar.gz>mp4p-${_mp4p_commit}.tar.gz"
+checksum="cbfe873d5bd84b70011dd6131b249fea8c89318c37411696a890c55029ac3e27
+ c5eeb3fa116d51d251fe7c7e5c44798680f6478c43a1fffdcb28c4caf1c68286
+ 35b58679dbfec2c3a2688b1896afb0fb73b5184ac05d5bdbb787a86b5c00430d"
+# clang is a build requirement as of v1.8.5
+build_options="clang gtk3"
+build_options_default="clang gtk3"
```

Comment:
I think we can drop both of these. `clang` is a requirement due to upstream using `-fblocks`, and we don't want GTK2 stuff.

You'd just have to do some slight clean up in `*depends` to remove gtk3 cleanly.

Reply:
```gtk3``` was there from the 1.8.3 template. I'll strip them both out and see if things still work.

Reply:
For the record, you want to strip out the `vopt_if gtk3` stuff as well. Otherwise the endeavor won't work :)

Reply:
Obviously ;)